engines/mxnet/mxnet-engine/src/main/java/ai/djl/mxnet/engine/MxSymbolBlock.java

```diff
@@ -184,9 +185,9 @@ protected NDList forwardInternal(
             NDList inputs,
             boolean training,
             PairList<String, Object> params) {
-        if (first) {
+        if (first.get()) {
             synchronized (MxSymbolBlock.class) {
-                if (first) {
+                if (first.get()) {
```

Comment:
Should this be `first.getAndSet(...)`?

Reply:
Let close this PR since this doesn't really work.